### PR TITLE
chore: Pass full repository name

### PR DESCRIPTION
**Pull request by michaeldowseza**
*Issue #, if available:*
Pass full repo name with owner.

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -67,5 +67,5 @@ jobs:
           AWSUI_GITHUB_BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
           AWSUI_GITHUB_COMMIT_SHA: ${{ github.event.commits[0].sha }}
           AWSUI_GITHUB_COMMIT_MESSAGE: ${{ github.event.commits[0].message }}
-          AWSUI_GITHUB_REPOSITORY_NAME: ${{ github.event.repository.name }}
+          AWSUI_GITHUB_REPOSITORY_NAME: ${{ github.event.repo.name }}
```
Comment (michaeldowseza) [inline .github/workflows/release.yml:70]: https://docs.github.com/en/rest/using-the-rest-api/github-event-types?apiVersion=2022-11-28

**Review by just-boris — APPROVED** [review]
ok, but how did `repository.name` work before https://github.com/cloudscape-design/actions/pull/35?

**Comment by michaeldowseza** [comment]
> ok, but how did `repository.name` worked before: [f2016f3](https://github.com/cloudscape-design/actions/commit/f2016f393421ed11f87b80e3f5a6d459f4f4da00)

It didn't. We relied on the env variables set. This is because I only tested locally and did not do an e2e dev branch test.

**Review by dpitcock — COMMENTED** [review]
Looks good

**Pull request merged**